#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-dialupadmin-help.dpatch by Mark Hymers <mark@hymers.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix README location

@DPATCH@

--- old/dialup_admin/htdocs/help/help.php3	(revision 1519)
+++ new/dialup_admin/htdocs/help/help.php3	(working copy)
@@ -48,13 +48,13 @@
 <?php
 $in_file = '';
 if ($help_file == 'readme')
-	$in_file = '../../README';
+	$in_file = '/usr/share/doc/freeradius-dialupadmin/README.gz';
 else if ($help_file == 'howto')
-	$in_file = '../../doc/HOWTO';
+	$in_file = '/usr/share/doc/freeradius-dialupadmin/HOWTO.gz';
 else if ($help_file == 'faq')
-	$in_file = '../../doc/FAQ';
+	$in_file = '/usr/share/doc/freeradius-dialupadmin/FAQ.gz';
 if ($in_file != '')
-	readfile("$in_file");
+	readgzfile("$in_file");
 ?>
 </pre>
 <br>
